mmv1/products/bigquerydatatransfer/Config.yaml

```diff
@@ -15,9 +15,8 @@
 name: 'Config'
 base_url: projects/{{project}}/locations/{{location}}/transferConfigs?serviceAccountName={{service_account_name}}
 self_link: '{{name}}'
-update_url: "{{name}}?serviceAccountName={{service_account_name}}"
+update_url: "{{name}}?serviceAccountName={{service_account_name}}&updateMask=serviceAccountName,displayName,destinationDatasetId,schedule,scheduleOptions,emailPreferences,notificationPubsubTopic,dataRefreshWindowDays,disabled,params"
```

Comment:
Can we use the update mask of `*` here instead of listing every field? It will make it difficult if we add new fields.

`*` update mask functionality is listed here: https://google.aip.dev/134

Reply:
when I tried setting `updateMask=*` I got this error:

```
Error: Error updating Config "-": googleapi: Error 400: * cannot be updated.
Details:
[
  {
    "@type": "type.googleapis.com/google.rpc.DebugInfo",
    "detail": "[ORIGINAL ERROR] generic::invalid_argument: * cannot be updated. [google.rpc.error_details_ext] { message: \"* cannot be updated.\" }"
  }
]
```

If this is the correct way of using `*` update mask, then maybe it's not supported by the API

Reply:
Hm, probably not supported by the API